.changelog/22972.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:bug
+resource/aws_ami_copy: Prevent panic when reading the source AMI's `boot_mode` or `destination` `outpost_arn`
+```
+
+```release-note:bug
+resource/aws_ami_from_instance: Prevent panic when reading the AMI's `boot_mode` or `destination` `outpost_arn`
+```
```

Comment:
As #22939 has not yet been released, these CHANGELOG entries should be enhancements:

```suggestion
```release-note:enhancement
resource/aws_ami_copy: Add `boot_mode` and `ebs_block_device.outpost_arn` attributes.
```

```release-note:enhancement
resource/aws_ami_from_instance: Add `boot_mode` and `ebs_block_device.outpost_arn` attributes.
```
```